Add Subscribe on AWS Marketplace section (#12)
forma-cloud/FormaCloud: README.md

7. Click "Add this app to a channel" then add the two FormaCloud channels respectively.
![image](https://github.com/forma-cloud/FormaCloud/assets/117554189/1ba7f5a7-564b-4121-9e91-c0e8fc3a7a6c)

## AWS Marketplace Subscription

> We are excited to inform you that our product is now available on the AWS Marketplace. This is a fantastic opportunity for you to access our products and services directly from the AWS Marketplace platform, and we ask you to please spend 2 minutes to subscribe.

Reviewer: ```suggestion

Our product is now available on the AWS Marketplace. Please spend 2 minutes to subscribe.
```

> By subscribing to us on the AWS Marketplace, **you are not going to pay anything extra**. In the near future, we will contact you to showcase our Optima solution, which will give you an effortless saving, and only if you agreed to install that solution will we charge a flat 10% from the savings we achieved.

Reviewer: ```suggestion
By subscribing to us on the AWS Marketplace, **you are not going to pay anything extra**. Only if you reviewed and agreed to start the Optima solution will we charge a flat 10% from the savings we achieved.
```
